cns/NetworkContainerContract.go

```diff
@@ -474,6 +474,8 @@ type PodIpInfo struct {
 	SkipDefaultRoutes bool
 	// Routes to configure on interface
 	Routes []Route
+	// PnpId is set for backend interfaces, Pnp Id identifies VF
+	PnPID string
```

Comment:
PnPID is being used by OS team, we want to be aligned on what os takes makes us of.

Reply:
Even if, in the code we want to align on internal terminology, it would be nice to have an "a.k.a." comment so it's possible to search for concepts in publicly-available documentation. From the outside, I will have no idea what a PnP is otherwise.

Reply:
Addressed